### ISSUE TITLE
Filter

### ISSUE THREAD

**Issue by sboddeus** [JensRavens/Interstellar#64]
Is there a reason other than lack of resources for not including `filter` on observables?
I would be happy to add a pull request unless I am missing a particular design decision.

**Comment by JensRavens** [comment]
The main reason so far is that I haven't had the need for it.

```swift
observable.subscribe {
  if condition {
    doWhatYouWant()
  }
}
```

But it makes sense to include it in `Observable`s directly (especially for mapping and chaining reasons). So if you want to go for it, PRs are welcome! :)

**Comment by sboddeus** [comment]
Here is my suggested implementation PR: https://github.com/JensRavens/Interstellar/pull/65

**Comment by JensRavens** [comment]
Closed by PR #65
